Mathlib/Topology/Maps.lean

```diff
@@ -649,23 +647,23 @@ theorem openEmbedding_id : OpenEmbedding (@id α) :=
   ⟨embedding_id, IsOpenMap.id.isOpen_range⟩
 #align open_embedding_id openEmbedding_id
 
-protected theorem OpenEmbedding.comp {g : β → γ} {f : α → β} (hg : OpenEmbedding g)
+protected theorem OpenEmbedding.comp (hg : OpenEmbedding g)
```

Comment:
Looks like the next few decls could also be organized in a `namespace OpenEmbedding` so that we can drop the `OpenEmbbeding` from all their names.

Reply:
There are only four such lemmas right now, so I wasn't sure if that would pull their weight. It doesn't seem to hurt either - changed.